Description: Upstream changes introduced in version 2.2.0-1
 This patch has been created by dpkg-source during the package build.
 Here's the last changelog entry, hopefully it gives details on why
 those changes were made:
 .
 slurm-llnl (2.2.0-1) unstable; urgency=low
 .
   * New upstream major release
   * libsurm22{,-dev} replace libsurm21{,-dev}
   * New libsurmdb22{,-dev} package
 .
 The person named in the Author field signed this changelog entry.
Author: Gennaro Oliva <oliva.g@na.icar.cnr.it>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- slurm-llnl-2.2.0.orig/doc/html/configurator.html.in
+++ slurm-llnl-2.2.0/doc/html/configurator.html.in
@@ -163,7 +163,7 @@ function displayfile()
    "#JobSubmitPlugins=1 <br>" +
    "#KillOnBadExit=0 <br>" +
    "#Licenses=foo*4,bar <br>" +
-   "#MailProg=/bin/mail <br>" +
+   "#MailProg=/usr/bin/mail <br>" +
    "#MaxJobCount=5000 <br>" +
    "#MaxTasksPerNode=128 <br>" +
    "MpiDefault=" + get_radio_value(document.config.mpi_default) + "<br>" +
--- slurm-llnl-2.2.0.orig/doc/man/man5/slurm.conf.5
+++ slurm-llnl-2.2.0/doc/man/man5/slurm.conf.5
@@ -722,7 +722,7 @@ not explicitly listed in the job submiss
 .TP
 \fBMailProg\fR
 Fully qualified pathname to the program used to send email per user request.
-The default value is "/bin/mail".
+The default value is "/usr/bin/mail".
 
 .TP
 \fBMaxJobCount\fR
@@ -1485,8 +1485,8 @@ thread on nodes with more than one \fBTh
 .TP
 \fBCR_CORE_DEFAULT_DIST_BLOCK\fR
 Allocate cores using block distribution by default.
-This default behavior can be overriden specifying a particular 
-"-m" parameter with srun/salloc/sbatch.
+This default behavior can be overridden specifying a particular 
+"\-m" parameter with srun/salloc/sbatch.
 Without this option, cores will be allocated cyclicly across the sockets.
 .TP
 \fBCR_Socket\fR
--- slurm-llnl-2.2.0.orig/doc/man/man5/slurmdbd.conf.5
+++ slurm-llnl-2.2.0/doc/man/man5/slurmdbd.conf.5
@@ -1,4 +1,4 @@
-.TH "slurmdbd.conf" "19" "August 2010" "slurmdbd.conf 2.2" "Slurm configuration file"
+.TH "slurmdbd.conf" "5" "August 2010" "slurmdbd.conf 2.2" "Slurm configuration file"
 .SH "NAME"
 slurmdbd.conf \- Slurm Database Daemon (SlurmDBD) configuration file
 
@@ -27,7 +27,9 @@ If ArchiveScript is not set the slurmdbd
 read in anytime with sacctmgr load filename.  This directory is where the
 file will be placed archive has ran.  Default is /tmp.  The format for this
 files name is
+.na
 $ArchiveDir/$ClusterName_$ArchiveObject_archive_$BeginTimeStamp_$endTimeStamp
+.ad
 
 .TP
 \fBArchiveEvents\fR
@@ -297,7 +299,9 @@ Acceptable values at present include
 "accounting_storage/pgsql".
 The value "accounting_storage/gold" indicates that account records
 will be written to Gold
+.na
 (http://www.clusterresources.com/pages/products/gold-allocation-manager.php),
+.ad
 which maintains its own database.
 The value "accounting_storage/mysql" indicates that accounting records
 should be written to a MySQL database specified by the
--- slurm-llnl-2.2.0.orig/doc/man/man5/cgroup.conf.5
+++ slurm-llnl-2.2.0/doc/man/man5/cgroup.conf.5
@@ -129,7 +129,7 @@ AllowedSwapSpace=10
 ConstrainCores=yes
 
 .SH "NOTES"
-Proctrack/cgroup is not compatable with the task/affinity plugin configured
+Proctrack/cgroup is not compatible with the task/affinity plugin configured
 to use cpusets, although this may be addressed in the future.
 .LP
 Only one instance of a cgroup subsystem is valid at a time in the kernel.
--- slurm-llnl-2.2.0.orig/doc/man/man1/strigger.1
+++ slurm-llnl-2.2.0/doc/man/man1/strigger.1
@@ -257,7 +257,7 @@ primary slurmctld fails.
 	strigger \-\-set \-\-primary_slurmctld_failure \\
 	         \-\-program=/usr/sbin/primary_slurmctld_failure
 	# Notify the administrator of the failure using by e\-mail
-	/bin/mail slurm_admin@site.com \-s Primary_SLURMCTLD_FAILURE
+	/usr/bin/mail slurm_admin@site.com \-s Primary_SLURMCTLD_FAILURE
 
 	> strigger \-\-set \-\-primary_slurmctld_failure \\
 	           \-\-program=/usr/sbin/primary_slurmctld_failure
@@ -276,7 +276,7 @@ argument to the script by SLURM).
 	strigger \-\-set \-\-node \-\-down \\
 	         \-\-program=/usr/sbin/slurm_admin_notify
 	# Notify administrator using by e\-mail
-	/bin/mail slurm_admin@site.com \-s NodesDown:$*
+	/usr/bin/mail slurm_admin@site.com \-s NodesDown:$*
 
 	> strigger \-\-set \-\-node \-\-down \\
 	           \-\-program=/usr/sbin/slurm_admin_notify
--- slurm-llnl-2.2.0.orig/doc/man/man1/sshare.1
+++ slurm-llnl-2.2.0/doc/man/man1/sshare.1
@@ -90,7 +90,7 @@ the user.  This number will decay over t
 is defined.
 
 .TP
-\f3Norm Usage\fP (only appears with \fBsshare -l\fR option)
+\f3Norm Usage\fP (only appears with \fBsshare \-l\fR option)
 The Raw Usage normalized to the total number of cpu-seconds of all
 jobs run on the cluster, subject to the PriorityDecayHalfLife decay
 when defined.
--- slurm-llnl-2.2.0.orig/doc/man/man1/sstat.1
+++ slurm-llnl-2.2.0/doc/man/man1/sstat.1
@@ -58,7 +58,7 @@ Predefined format to list the pids runni
 \f3\-j\fP\f3,\fP \f3\-\-jobs\fP
 Format is <job(.step)>. Stat this job step or comma-separated list of
 job steps. This option is required.  The step portion will default to
-lowest step running if not specified, unless the --allsteps flag is
+lowest step running if not specified, unless the \-\-allsteps flag is
 set where not specifying a step will result in all running steps to be
 displayed.
 NOTE: A step id of 'batch' will display the information about the batch step.
--- slurm-llnl-2.2.0.orig/doc/man/man3/slurm_update_job.3
+++ slurm-llnl-2.2.0/doc/man/man3/slurm_update_job.3
@@ -29,7 +29,7 @@ int \fBslurm_update_job\fR (
 .LP
 int \fBslurm_update_step\fR (
 .br
-	step_update_request_msg_t * \step_msg\fP
+	step_update_request_msg_t * \fIstep_msg\fP
 .br
 );
 
--- slurm-llnl-2.2.0.orig/src/common/read_config.h
+++ slurm-llnl-2.2.0/src/common/read_config.h
@@ -84,7 +84,7 @@ extern char *default_plugstack;
 #define DEFAULT_KILL_ON_BAD_EXIT    0
 #define DEFAULT_KILL_TREE           0
 #define DEFAULT_KILL_WAIT           30
-#define DEFAULT_MAIL_PROG           "/bin/mail"
+#define DEFAULT_MAIL_PROG           "/usr/bin/mail"
 #define DEFAULT_MAX_JOB_COUNT       10000
 #define DEFAULT_MEM_PER_CPU         0
 #define DEFAULT_MAX_MEM_PER_CPU     0
--- slurm-llnl-2.2.0.orig/src/slurmctld/step_mgr.c
+++ slurm-llnl-2.2.0/src/slurmctld/step_mgr.c
@@ -914,7 +914,7 @@ _pick_step_nodes (struct job_record  *jo
 					i : step_spec->min_nodes ;
 		if (step_spec->max_nodes &&
 		    (step_spec->max_nodes < step_spec->min_nodes)) {
-			info("Job step %u max node count incompatable with CPU "
+			info("Job step %u max node count incompatible with CPU "
 			     "count", job_ptr->job_id);
 			*return_code = ESLURM_TOO_MANY_REQUESTED_CPUS;
 			goto cleanup;
--- slurm-llnl-2.2.0.orig/src/plugins/proctrack/cgroup/xcgroup.c
+++ slurm-llnl-2.2.0/src/plugins/proctrack/cgroup/xcgroup.c
@@ -526,7 +526,7 @@ int xcgroup_set_params(char* cpath,char*
 				       p,value,cpath);
 		}
 		else
-			debug2("bad paramters format for entry '%s'",p);
+			debug2("bad parameters format for entry '%s'",p);
 	next_loop:
 		p = next;
 	}
